app/javascript/react/containers/SuperpowerShowContainer.js

```diff
@@ -20,21 +22,42 @@ class SuperpowerShowContainer extends React.Component {
     .then(json => {
       this.setState({
         name: json.superpower.name,
-        description: json.superpower.description
+        description: json.superpower.description,
+        reviews: json.reviews
       })
     })
   }
 
+  addNewReview(formPayload) {
+    let superpowerId = this.props.params.id
+    fetch(`/api/v1/superpowers/${superpowerId}/reviews`, {
+      credentials: 'same-origin',
+      header: {
+       'Content-Type': 'application/json',
+       'X-Requested-With': 'XMLHttpRequest'
```

Comment:
I don't believe you need this so long as you have the `skip_before_action :verify_authenticity_token` in your ApiController!